### PR TITLE
Removing last ARG_STRING reference

### LINKED ISSUE #263: Outdated README directions in /examples/static_react_task/README.md
In the last section, **Making your own static react task**

it says:

```3. Update the task-related `ARG_STRING` variables in `run_task.py` to make sense for your task.```

which is outdated now that we don't use `ARG_STRING` anymore in favor of hydra.

### PR DESCRIPTION
**Pull request by JackUrb**
Title

closes #263

### CODE REVIEW

File: examples/static_react_task/README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ In order to get started on your own task, it is a good idea to copy this `static
 
 1. Copy the `static_react_task` directory to your project directory
 2. Update `static_task_data` in the `run_task.py` script to provide the required data for your task
-3. Update the task-related `ARG_STRING` variables in `run_task.py` to make sense for your task.
+3. Update the any task-related variables in your `conf/my_new_config.yaml` file to make sense for your task.
```
Comment (pringshia) [inline examples/static_react_task/README.md:109]: small typo: "the any"

File: examples/static_react_task/README.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ In order to get started on your own task, it is a good idea to copy this `static
 
 1. Copy the `static_react_task` directory to your project directory
 2. Update `static_task_data` in the `run_task.py` script to provide the required data for your task
-3. Update the task-related `ARG_STRING` variables in `run_task.py` to make sense for your task.
+3. Update anytask-related variables in your `conf/my_new_config.yaml` file to make sense for your task.
```
Comment (pringshia) [inline examples/static_react_task/README.md:109]: missing a space now 😬... otherwise looks good!

Comment (JackUrb) [reply examples/static_react_task/README.md:109]: This is what I get for a find-replace. Thanks haha

Comment (JackUrb) [reply examples/static_react_task/README.md:109]: Oh wait this is what I get for not saving a file :(

**Pull request merged**